README.md

```diff
@@ -1,3 +1,48 @@
 [![Build Status](https://api.travis-ci.org/alphagov/paas-cf.svg)](https://travis-ci.org/alphagov/paas-cf)
 
 # paas-cf
+
+This repository contains [Concourse](http://concourse.ci/) pipelines and related [Terraform](https://terraform.io/) and [BOSH](https://bosh.io/) manifests that allow provisioning of [CloudFoundy](https://www.cloudfoundry.org/) on AWS. It consists of two chains:
+- A concourse pipeline to provision CloudFoundry on AWS using Concourse, Terraform and BOSH
+- A concourse pipeline to destroy previously provisioned environment.
+
+## Operation
+
+* A local Vagrant virtual machine is provisioned with concourse-lite.
+* The deployment pipeline is pushed to concourse-lite using the concourse `fly` command.
```

Comment:
Well, first it's not that detailed - it's just an overview. Second to move it next to scripts it would be good to make it more detailed or even merge with something like https://github.com/alphagov/paas-cf/pull/6#discussion_r47348696